server/command.go

```diff
@@ -246,3 +247,25 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, "Unknown action %v", action)
 	return &model.CommandResponse{}, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	welcomebot := model.NewAutocompleteData("welcomebot", "[command]",
+		"Available commands: preview, help, list, set_channel_welcome, get_channel_welcome, delete_channel_welcome")
+
+	preview := model.NewAutocompleteData("preview", "[team-name]", "Preview the welcome message for the given team name")
+	welcomebot.AddCommand(preview)
+
+	list := model.NewAutocompleteData("list", "", "Lists team welcome messages")
+	welcomebot.AddCommand(list)
+
+	setChannelWelcome := model.NewAutocompleteData("set_channel_welcome", "[welcome-message]", "Set the welcome message for the channel")
```

Comment:
Same here for `welcome-message`

Reply:
Committed at https://github.com/mattermost/mattermost-plugin-welcomebot/pull/69/commits/ffeadbc340f17b1c44c62fa9f7c586b85aadb859